examples/01-filter/glyphs_table.py

```diff
@@ -39,8 +39,8 @@
 geoms = [pv.ParametricSuperToroid(n1=n1, n2=n2) for n1, n2 in params]
 
 # get dataset where to put glyphs
-x,y,z = np.mgrid[:3, :3, :3]
-mesh = pv.StructuredGrid(x, y, z)
+x, y, z = np.mgrid[:3, :3, :3]
+mesh = pv.StructuredGrid(x, y, z, force_float=False)
```

Comment:
Same as elsewhere: I'd prefer
```suggestion
x, y, z = np.mgrid[:3.0, :3.0, :3.0]
mesh = pv.StructuredGrid(x, y, z)
```
assuming the kwarg was there to prevent the warning.